docs/docs/noir/concepts/generics.md

```diff
@@ -104,3 +104,60 @@ impl Eq for MyStruct {
 ```
 
 You can find more details on traits and trait implementations on the [traits page](../concepts/traits.md).
+
+## Manually Specifying Generics with the Turbofish Operator
+
+There are times when the compiler cannot reasonably infer what type should be used for a generic, or when the developer themselves may want to manually distinguish generic type parameters. This is where the `::<>` turbofish operator comes into play.
+
+Paths with generic parameters in expressions can prefix the opening brackets with a `::<>` operator. They must be prefixed with `::<>` in cases where the generic type cannot be inferred.
```

Comment:
```suggestion
The `::<>` operator can follow a variable or path and can be used to manually specify generic arguments within the angle brackets.
```
I found the original sentence to be a bit awkwardly worded. Otherwise the PR looks good